Tasks/SqlAzureDataWarehouseDacpacDeploymentV0/task.loc.json

```diff
@@ -0,0 +1,356 @@
+{
+  "id": "2D429209-58FB-4A53-83AD-9CF58807B56C",
+  "name": "SqlAzureDataWarehouseDacpacDeployment",
+  "friendlyName": "ms-resource:loc.friendlyName",
+  "description": "ms-resource:loc.description",
+  "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/deploy/sql-azure-dacpac-deployment",
+  "helpMarkDown": "ms-resource:loc.helpMarkDown",
+  "category": "Deploy",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "runsOn": [
+    "Agent"
+  ],
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 1,
+    "Minor": 156,
+    "Patch": 0
+  },
+  "demands": [
+    "sqlpackage"
+  ],
+  "minimumAgentVersion": "1.103.0",
+  "groups": [
+    {
+      "name": "target",
+      "displayName": "ms-resource:loc.group.displayName.target",
+      "isExpanded": true
+    },
+    {
+      "name": "taskDetails",
+      "displayName": "ms-resource:loc.group.displayName.taskDetails",
+      "isExpanded": true
+    },
+    {
+      "name": "firewall",
+      "displayName": "ms-resource:loc.group.displayName.firewall",
+      "isExpanded": false
+    }
+  ],
+  "inputs": [
+    {
+      "name": "ConnectedServiceNameSelector",
+      "aliases": [
+        "azureConnectionType"
+      ],
+      "type": "pickList",
+      "label": "ms-resource:loc.input.label.ConnectedServiceNameSelector",
+      "required": false,
+      "helpMarkDown": "",
+      "defaultValue": "ConnectedServiceNameARM",
+      "options": {
+        "ConnectedServiceName": "Azure Classic",
```

Comment:
Since this is a new task. I think we don't need to support Azure classic endpoint to the task.

Reply:
Do you want me to only remove this from the task.loc.json and task.json files? or you want me to go through all the code and remove it from everywhere? This could be a little bit risky since we are targeting the next release.